great_expectations/data_context/data_context/cloud_data_context.py

```diff
@@ -113,45 +134,42 @@ def _register_providers(self, config_provider: _ConfigurationProvider) -> None:
         """
         super()._register_providers(config_provider)
         config_provider.register_provider(
-            _CloudConfigurationProvider(self._ge_cloud_config)
+            _CloudConfigurationProvider(self._cloud_config)
         )
 
     @classmethod
-    def is_ge_cloud_config_available(
+    def is_cloud_config_available(
```

Comment:
I've renamed a few class methods that are "public" but should never be used by a user - if this is an issue, please let me know and we can revert.